config/application.rb

```diff
@@ -16,5 +16,6 @@ class Application < Rails::Application
                          { expires_in: 90.minutes }
     config.active_job.queue_adapter = :delayed_job
     config.active_record.schema_format = :ruby
```

Comment:
I recommend to use `:sql` (`structure.sql`), you know it :)

Reply:
added it in sequel branch